sphinx/user/getting_started.rst

```diff
@@ -231,12 +232,18 @@ Now for using the pipeline.
       
      constructFiberTrace.py $PFS_DATA_DIR --rerun $whoami/calibs --id visit=5 --calibId calibVersion=fiberTrace calibDate=2015-12-22 arm=r spectrograph=2 -c isr.doBias=True isr.doDark=True isr.doFlat=False isr.doLinearize=False --batch-type none
      genCalibRegistry.py --root $PFS_DATA/CALIB --camera PFS --validity 180
+
+- We can now construct our master Flat from the dithered Flats, which have the
+  visit numbers 29, 41, 42, 44, 45, 46, 47, 48, 49, 51, and 53. Since these are
+  the only visit numbers between 29 and 53 we can just specify 'visit=29..53'::
+      
+     constructFiberFlat.py $PFS_DATA --rerun $whoami/tmp --id visit=29..53 dateObs=2015-12-22 arm=r spectrograph=2 --calibId calibVersion=flat calibDate=2015-12-22 arm=r spectrograph=2 --batch-type none
+     genCalibRegistry.py --root $PFS_DATA/CALIB --camera PFS --validity 180
      
-- Since we have the master Bias and Dark we can now perform the
+- Since we have the master Bias, Dark, and Flat we can now perform the
   Instrumental-Signature Removal (ISR) task for our Arc spectrum (visit=4).
   The program detrend.py will start the ISR task which will subtract the Bias
-  and scaled Dark from our Arc image. Flat-fielding is not yet supported by the
-  pipeline, but will be in the near future.
+  and scaled Dark from our Arc image.
 
   If you want to reduce all Arcs taken 2015-12-22 for spectrograph 2, red arm,
   simply replace ``visit=4`` with ``arm=r spectrograph=2 dateObs=2015-12-22
```

Comment:
Is `visit=4` correct with the new stella_test_data?

The startup document **must** be correct!